project-loader/src/main/kotlin/de/itemis/mps/gradle/project/loader/PluginIds.kt

```diff
@@ -0,0 +1,126 @@
+package de.itemis.mps.gradle.project.loader
+
+import de.itemis.mps.gradle.logging.detectLogging
+import org.w3c.dom.Document
+import java.io.File
+import java.io.IOException
+import java.io.InputStream
+import java.nio.file.FileVisitResult
+import java.nio.file.Files
+import java.nio.file.Path
+import java.nio.file.SimpleFileVisitor
+import java.nio.file.attribute.BasicFileAttributes
+import java.util.jar.JarFile
+import javax.xml.parsers.DocumentBuilder
+import javax.xml.parsers.DocumentBuilderFactory
+import kotlin.io.path.absolutePathString
+import kotlin.io.path.isDirectory
+
+private val logger = detectLogging().getLogger("de.itemis.mps.gradle.tasks.PluginIds")
```

Comment:
The logger name should be adjusted:
```suggestion
private val logger = detectLogging().getLogger("de.itemis.mps.gradle.project.loader.PluginIds")
```